src/views/tutorial-library/components/current-filters/index.tsx

```diff
@@ -0,0 +1,105 @@
+import { useCurrentRefinements } from 'react-instantsearch-hooks-web'
+import Text from 'components/text'
+import Tag from 'components/tag'
+import { productSlugsToNames } from 'lib/products'
+import currentFiltersStyle from './current-filters.module.css'
+import { EDITIONS, RESOURCES } from '../../constants'
+import { ClearFiltersButton } from '../clear-filters-button'
+
+interface Refinement {
+	label: string
+	type: string
+	attribute: string
+}
+
+interface CurrentFilterItemProps {
+	refinement: Refinement
+	refine: (refinement: Refinement) => void
+}
+
+function CurrentFilterItem({ refinement, refine }: CurrentFilterItemProps) {
+	const { label, type, attribute } = refinement
+
+	let labelText = label
+	// This is a "Resource" filter
+	if (type === 'disjunctive') {
+		const resource = RESOURCES.find(
+			(resource) => resource.attribute === attribute
+		)
+		labelText = resource.label
+	}
+
+	if (attribute === 'edition') {
+		const edition = EDITIONS.find((edition) => edition.value === label)
+		labelText = edition.label
+	}
+
+	if (attribute === 'products') {
+		labelText = productSlugsToNames[label]
+	}
+
+	return (
+		<li>
+			{/* TODO: test with voice control, validate that it fulfills Success Criterion 2.5.3 Label in Name */}
```

Comment:
Curious if we tested this with voice control here? Or if the TODO is still valid

Reply:
I personally have not yet.